dev/NavigationView/NavigationView.cpp

```diff
@@ -1917,12 +1908,13 @@ void NavigationView::UpdatePaneTitleFrameworkElementParents()
         const auto isPaneToggleButtonVisible = IsPaneToggleButtonVisible();
         const auto isTopNavigationView = IsTopNavigationView();
 
-        paneTitleHolderFrameworkElement.Visibility(
-            (isPaneToggleButtonVisible ||
-                isTopNavigationView ||
-                PaneTitle().size() == 0 ||
-                (PaneDisplayMode() == winrt::NavigationViewPaneDisplayMode::LeftMinimal && !IsPaneOpen())) ?
-            winrt::Visibility::Collapsed : winrt::Visibility::Visible);
+        m_isPaneTitleEmpty = (isPaneToggleButtonVisible ||
+            isTopNavigationView ||
+            PaneTitle().size() == 0 ||
+            (PaneDisplayMode() == winrt::NavigationViewPaneDisplayMode::LeftMinimal && !IsPaneOpen()));
+
+        paneTitleHolderFrameworkElement.Visibility(m_isPaneTitleEmpty ? winrt::Visibility::Collapsed : winrt::Visibility::Visible);
```

Comment:
Might be better to have visual state then setting visibility in .cpp? :)

Reply:
unfortunately, there is no visual state explicitly for when the panetitle is empty :( As this is a small change, I think it is better to leave it as is then to refactor this part.